packages/core/src/physics/DynamicCollider.ts

```
@@ -270,13 +270,13 @@ export class DynamicCollider extends Collider {
 
   set isKinematic(value: boolean) {
     if (this._isKinematic !== value) {
-      // Block switching to non-kinematic if triangle mesh is attached
       if (!value) {
+        // Block switching to non-kinematic if non-convex MeshColliderShape is attached
         const shapes = this._shapes;
         for (let i = 0, n = shapes.length; i < n; i++) {
           const shape = shapes[i];
           if (shape instanceof MeshColliderShape && !shape.isConvex) {
-            console.error("DynamicCollider: Cannot set isKinematic=false when triangle mesh is attached.");
+            console.error("DynamicCollider: Cannot set isKinematic=false when non-convex MeshColliderShape is attached.");
```

ESLint

[error] Replace `"DynamicCollider:·Cannot·set·isKinematic=false·when·non-convex·MeshColliderShape·is·attached."` with `⏎··············"DynamicCollider:·Cannot·set·isKinematic=false·when·non-convex·MeshColliderShape·is·attached."⏎············`

(prettier/prettier)